### PR TITLE
Fb update pr template

### PR DESCRIPTION
**Pull request by aliabdolali**
# Pull Request Summary
This PR updates the PR template with instructions to add labels and commit message summary.

## Description
In the checklist, the developers are asked to add labels and a short summary of the commit messages to the repo managers, to have clean log message.

### Check list  
* Is your feature branch up to date with the authoritative repository (NOAA/develop)?
Yes
* Reviewers: @JessicaMeixner-NOAA 

### Testing
* How were these changes tested?
No
* Are the changes covered by regression tests? (If not, why? Do new tests need to be added?)
NA
No need to run the regtest.

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -20,8 +20,12 @@ Note that properly "linked issues" (either automatic links, or manual ones using
 ### Check list  
 * Is your feature branch up to date with the authoritative repository (NOAA/develop)?
 * Make sure you have checked the [checklist for a developer submitting to develop](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-a-developer-submitting-to-develop) and [updating version number](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-updating-version-number)
+* Make sure you have added labels to the PR, including but not limited to _bug_, _documentation_, _duplicate_, _enhancement_, _invalid_, _mod_def change_, _out_grd change_, _out_pnt change_, _restart file change_, _new feature_, _Regression test_, _UFS_, ...
```
Comment (JessicaMeixner-NOAA) [inline .github/pull_request_template.md]: Before adding this, we should check that someone without write permissions can in fact add labels.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -20,8 +20,12 @@ Note that properly "linked issues" (either automatic links, or manual ones using
...
+
 * Reviewers: @mentions of suggested reviewers of the proposed changes
 
+### Commit Message
```
Comment (JessicaMeixner-NOAA) [inline .github/pull_request_template.md:28]: Log Message -> the git log message

File: .github/pull_request_template.md (hunk 1 above)
Comment (JessicaMeixner-NOAA) [reply .github/pull_request_template.md]: the changes could be questions in the regression tests part as that's more closely linked to that.

Comment (aliabdolali) [reply .github/pull_request_template.md]: moved there

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -31,5 +34,5 @@ Note that properly "linked issues" (either automatic links, or manual ones using
 * Which compiler / HPC you used to run the regression tests in the PR? 
 * Please provide the summary output of matrix.comp (_matrix.Diff.txt_, _matrixCompFull.txt_ and _matrixCompSummary.txt_):    
 Please indicate the expected changes in the outputs ([excluding the known list of non-identical tests](https://github.com/NOAA-EMC/WW3/wiki/How-to-use-matrix.comp-to-compare-regtests-with-master#4-look-at-results)).
-
+* Make sure you have added labels to the PR, including but not limited to _bug_, _documentation_, _duplicate_, _enhancement_, _invalid_, _mod_def change_, _out_grd change_, _out_pnt change_, _restart file change_, _new feature_, _Regression test_, _UFS_, ...
```
Comment (JessicaMeixner-NOAA) [inline .github/pull_request_template.md]: So perhaps this should be, please add appropriate labels to indicate a change: 
_mod_def change_, _out_grd change_, _out_pnt change_, _restart file change_


And then maybe somewhere else we ask for labels such as _bug_ or _documentation_,  _new feature_   ?   Some of these labels were created for issues and some fit but some seem to be duplicates of each other  and maybe not all are necessary.  Hopefully someone isn't creating a new invalid PR for example.

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -31,5 +36,6 @@ Note that properly "linked issues" (either automatic links, or manual ones using
 * Which compiler / HPC you used to run the regression tests in the PR? 
 * Please provide the summary output of matrix.comp (_matrix.Diff.txt_, _matrixCompFull.txt_ and _matrixCompSummary.txt_):    
 Please indicate the expected changes in the outputs ([excluding the known list of non-identical tests](https://github.com/NOAA-EMC/WW3/wiki/How-to-use-matrix.comp-to-compare-regtests-with-master#4-look-at-results)).
-
+* please add appropriate labels to indicate a change (The labels should be added by the code managemers):
```
Comment (JessicaMeixner-NOAA) [inline .github/pull_request_template.md]: Maybe: 
Please list which labels code managers should add to indicate code changes:  _mod_def change_, _out_grd change_, _out_pnt change_, _restart file change_, ...

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -20,8 +20,13 @@ Note that properly "linked issues" (either automatic links, or manual ones using
 ### Check list  
 * Is your feature branch up to date with the authoritative repository (NOAA/develop)?
 * Make sure you have checked the [checklist for a developer submitting to develop](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-a-developer-submitting-to-develop) and [updating version number](https://github.com/NOAA-EMC/WW3/wiki/Code-Management#checklist-for-updating-version-number)
+* please add appropriate labels to indicate a change (The labels should be added by the code managemers):
```
Comment (JessicaMeixner-NOAA) [inline .github/pull_request_template.md]: Maybe: 
Please list appropriate labels code managers should add for this PR: _bug_, _documentation_, _enhancement_, _new feature_, ....

**Review by JessicaMeixner-NOAA — APPROVED** [review]

**Pull request merged**